pages/api/snippets/index.js

```diff
@@ -0,0 +1,10 @@
+import dbConnect from "@/db/connect";
+import Snippet from "@/db/models/Snippet";
+
+export default async function handler(request, response) {
+  await dbConnect();
```

Comment:
you can consider wrapping all of this in a try-catch block, in case something goes wrong